helm/vitess/templates/_orchestrator-conf.tpl

```diff
@@ -35,13 +35,14 @@ data:
     "DetectClusterDomainQuery": "",
     "DetectInstanceAliasQuery": "SELECT value FROM _vt.local_metadata WHERE name='Alias'",
     "DetectPromotionRuleQuery": "SELECT value FROM _vt.local_metadata WHERE name='PromotionRule'",
+    "DetectDataCenterQuery": "SELECT value FROM _vt.local_metadata WHERE name='DataCenter'",
     "DetectPseudoGTIDQuery": "",
     "DetectSemiSyncEnforcedQuery": "SELECT @@global.rpl_semi_sync_master_wait_no_slave AND @@global.rpl_semi_sync_master_timeout > 1000000",
-    "DiscoverByShowSlaveHosts": true,
+    "DiscoverByShowSlaveHosts": false,
     "EnableSyslog": false,
     "ExpiryHostnameResolvesMinutes": 60,
-    "FailMasterPromotionIfSQLThreadNotUpToDate": true,
-    "FailureDetectionPeriodBlockMinutes": 60,
+    "DelayMasterPromotionIfSQLThreadNotUpToDate": true,
```

Comment:
This is fine if working, but should eventually turn into an option. For our use cases, I would still prefer to fail rather than just delay master promotion. I think it's workload specific.

Reply:
You may misunderstand this: setting this value to false does not mean that the promotion will fail. It means that a replica will be promoted as master without applying all the relay log events, essentially losing data and causing split brain.

Reply:
He is removing `FailMasterPromotionIfSQLThreadNotUpToDate` in this change in favor of `DelayMasterPromotionIfSQLThreadNotUpToDate`. I'm saying I prefer the original setting that theoretically avoids split brain, but isn't as HA friendly.

Reply:
Ahh. Missed that part.